cpmonitor/views.py

```diff
@@ -277,32 +298,15 @@ def cap_checklist_view(request, city_slug):
     return render(request, "cap_checklist.html", context)
 
 
-def _get_cap_checklist(city) -> dict:
+def _get_cap_checklist(city) -> list:
     try:
         checklist = city.cap_checklist
     except CapChecklist.DoesNotExist:
-        return {}
+        return []
 
     return _as_formatted_checklist(checklist)
 
 
-def _as_formatted_checklist(checklist):
-    checkbox_items = [
-        field
-        for field in checklist._meta.get_fields()
-        if field.attname not in ["city_id", "id"] and "_rationale" not in field.attname
-    ]
-
-    return {
-        checkbox_item.verbose_name: {
-            "is_checked": getattr(checklist, checkbox_item.attname),
-            "help_text": checkbox_item.help_text,
-            "rationale": getattr(checklist, checkbox_item.attname + "_rationale"),
-        }
-        for checkbox_item in checkbox_items
-    }
```

Comment:
Are you really sure that the other two checklists still work? In my mind a voice says that there was a good reason to work with dict and not list because we need the `checkbox_item.verbose_name` from line 297 but if it still works all good 😉

Reply:
They look fine to me. I also adapted the html of course so that it uses checklist_item.question now instead of the dictionary key, but as it anyway looped over all dictionary entries, that was no problem.